### DISCUSSION TITLE
Release v1.14.6 / 5.69.6

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4132] [Announcements]
This build fixes various bugs and adds some minor improvements.

For a full list of changes and fixes please review the [full change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1146--5696---2024-07-30).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.6'>Release v1.14.6 / 5.69.6</a>.</em>

**Comment by jiangnancao** [comment]
software   https://wwp.lanzouo.com/jsm-win    |   Version 1.14.1 can be used, but the new version Sandboxie Plus cannot be used

**Reply by isaak654** [reply]
Please open a new [problem report](https://github.com/sandboxie-plus/Sandboxie/issues/new/choose).
